src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -641,8 +635,7 @@ private static boolean isEmptyType(DetailAST ast) {
         final DetailAST previousSibling = ast.getPreviousSibling();
         return type == TokenTypes.LCURLY
                     && nextSibling.getType() == TokenTypes.RCURLY
-                || type == TokenTypes.RCURLY
```

Comment:
No way to find something that wasn't a rightcurly and the previous sibling was a left curly.